src/routers/bookRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const {getBooksById, getBooksByName} = require('../controllers/bookController')
+
+
+const router = express.Router();
+
+router.get("/search", getBooksByName);
+router.get("/sub_categories/:id", getBooksById);
```

Comment:
RESTful API에서는 _ 언더바 대신 - 하이픈을 쓰는 것을 권장합니다.

만약 누군가에게 우리 서비스의 url을 전달하는 경우, 링크 자체를 전달하면서 자동으로 밑줄이 생기는 현상을 본 적이 있을 것입니다.

_ 언더바가 포함되면 자동으로 생성된 밑줄과의 혼선이 생길 수 있어 엔드포인트에는 작성하지 않는 것이 좋겠습니다.

books의 id 값으로 특정 리소스를 조회하는 것이기 때문에 http://localhost:3000/books/1 과 같은 엔드포인트가 가장 적절해보입니다.

여기서는 `getBooksByCategoryId`라는 함수명이 좋을 것 같네요!
```suggestion
router.get("/subCategories/:id", getBooksByCategoryId);
```

Reply:
수정 완료하였습니다!